{"name": "test_login", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('self', <FixtureRequest for <Function test_login>>)", "trace": "file C:\\Python_Selenium\\test_fixture.py, line 7\n  @pytest.mark.sanity\n  def test_login(self):\nE       fixture 'self' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, launch, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nC:\\Python_Selenium\\test_fixture.py:7"}, "start": 1743159924995, "stop": 1743159924995, "uuid": "bf311558-cd10-49ee-ad5f-5c1c05ef5905", "historyId": "c14209f77d6be8ab031e6039a01375e6", "testCaseId": "c14209f77d6be8ab031e6039a01375e6", "fullName": "test_fixture#test_login", "labels": [{"name": "tag", "value": "sanity"}, {"name": "suite", "value": "test_fixture"}, {"name": "host", "value": "DESKTOP-LIFTNJ8"}, {"name": "thread", "value": "34296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_fixture"}]}